android/app/build.gradle

```diff
@@ -111,12 +116,12 @@ dependencies {
     implementation 'androidx.navigation:navigation-ui-ktx:2.3.0'
     implementation 'androidx.recyclerview:recyclerview:1.1.0'
     implementation "androidx.webkit:webkit:1.3.0"
-    implementation "androidx.preference:preference:1.1.1"
+    implementation "androidx.preference:preference-ktx:1.1.1"
     implementation 'com.google.android.material:material:1.2.0'
     implementation 'org.slf4j:slf4j-api:1.7.30'
-    implementation 'com.synnapps:carouselview:0.1.5'
+    implementation 'com.synnapps:carouselview:0.1.6'
     implementation 'com.google.code.gson:gson:2.8.9'
-    implementation group: 'org.apache.jena', name: 'jena-core', version: '3.17.0'
+    implementation group: 'org.apache.jena', name: 'jena-core', version: '4.2.0'
```

Comment:
Isn't that including https://github.com/polypoly-eu/polyPod/pull/420 -> Which is not ready to land?

Reply:
Right, and it's not because it needed the move from 8 to 11, which I'm doing here and apparently works. I just inserted it here to see if it failed... and I couldn't make it fail in Android studio. My hunch is that the Gradle upgrade took care of that, but I might be wrong. You might want to run the same tests you did for #420 (a very proper number for an upgrade to 4.2.0), but it would be great if those tests were also included here.

Reply:
Well, all it needs to _test_ is to actually launch the app on a simulator with our oldest supported Android version (I think it was API level 26?)

Reply:
The thing is that that variable has not changed, it still says the same. Maybe I'm wrong, but I have the impression that the newer gradle version is able to transpile to that minSDKversion indicated in the gradle file. We need to check anyway, of course, but the fact that gradle yields apparently no error or warning to the effect of the min SDK version being incompatible with the Java version makes me hopeful.